apis/v1alpha2/shared_types.go

```diff
@@ -331,10 +333,9 @@ type RouteStatus struct {
 // Hostname is the fully qualified domain name of a network host. This matches
 // the RFC 1123 definition of a hostname with 2 notable exceptions:
 //
-// 1. IPs are not allowed.
-// 2. A hostname may be prefixed with a wildcard label (`*.`). The wildcard
-//
-//	label must appear by itself as the first label.
+//  1. IPs are not allowed.
+//  2. A hostname may be prefixed with a wildcard label (`*.`). The wildcard
+//     label must appear by itself as the first label.
```

Comment:
Why the extra spaces here? If I'm understanding this correctly, we're moving from one space after `//` to two, is that intentional?

Reply:
It was unintentional, `go fmt` somehow produced that. I noticed that the "verify" job failed. I have updated the `yaml` files, should be good now.